praw/models/listing/mixins/base.py

```diff
@@ -6,7 +6,10 @@
 
 def _prepare(praw_object, arguments_dict, target):
     """Fix for Redditor methods that use a query param rather than subpath."""
-    if praw_object.__dict__.get("_listing_use_sort"):
+    if (
```

Comment:
Would a try/except be better here now?

Reply:
It wouldn’t be faster to do so. Raising exceptions is still usually slower than doing `if` checks.

The `try`/`except` approach is only faster if the assumption of the key existing holds true in a majority of cases. It won’t be here.